lms/resources/_js_config/file_picker_config.py

```diff
@@ -4,68 +4,67 @@ class FilePickerConfig:
     @classmethod
     def blackboard_config(cls, context, request, application_instance):
         """Get Blackboard files config."""
-        if not application_instance.product == application_instance.Product.BLACKBOARD:
-            return {}
-
-        enabled = application_instance.settings.get("blackboard", "files_enabled")
+        files_enabled = application_instance.settings.get("blackboard", "files_enabled")
+        groups_enabled = context.blackboard_groups_enabled
 
         auth_url = request.route_url("blackboard_api.oauth.authorize")
         course_id = request.params.get("context_id")
 
-        return {
-            "enabled": enabled,
-            "groupsEnabled": context.blackboard_groups_enabled,
-            "listFiles": {
+        config = {
+            "enabled": files_enabled,
```

Comment:
Can't change the key here without changing it on the frontend. We probably should but in another reafactor PR.